src/domain/events/detail/EventShow.tsx

```diff
@@ -175,18 +176,25 @@ const EventShow = (props: ResourceComponentPropsWithId) => {
             source={`duration`}
             label={'events.fields.duration.label'}
           />
-          {internalTicketSystem ? (
+          {internalTicketSystem && (
             <NumberField
               source="capacityPerOccurrence"
               label="events.fields.capacityPerOccurrence.label"
             />
-          ) : (
+          )}
+          {!internalTicketSystem && (
             <SelectField
               source="ticketSystem.type"
               label="events.fields.ticketSystem.label"
               choices={ticketSystemChoices}
             />
           )}
+          {!internalTicketSystem && (
```

Comment:
the type and the url could be handled in a single if-statement, so there would be 2 different clear paths, instead of "multiple ones". It would then be more like if-else-statement.